.github/scripts/trymerge.py

```diff
@@ -1055,18 +1089,44 @@ def get_commit_next_checksuites(checksuites: Any) -> Any:
 def checks_to_str(checks: List[Tuple[str, Optional[str]]]) -> str:
     return ", ".join(f"[{c[0]}]({c[1]})" if c[1] is not None else c[0] for c in checks)
 
-def pr_get_checks_with_lambda(pr: GitHubPR, status_check: Callable[[Optional[str]], bool]) -> List[Tuple[str, str]]:
-    checks = pr.get_checkrun_conclusions()
-    return [(name, status[1]) for name, status in checks.items() if status_check(status[0])]
+def get_combined_checks_from_pr_and_land_validation(
+    pr: GitHubPR,
+    land_check_commit: Optional[str]
+) -> Dict[str, WorkflowCheckState]:
+    """
```

Comment:
This feels like a weird way to leave a comment. Do we do this anywhere else on this codebase?

Reply:
This is the official way to write documentation for python function (called a 'docstring'), and yes, we have many examples of it :)

It def feels weird when we're not used to it though